understanding/meaningful-sequence.html

```diff
@@ -35,7 +35,7 @@ <h1>達成基準 1.3.2: 意味のあるシーケンスを理解する</h1>
             <h2>意図</h2>
             <p>この達成基準の意図は、コンテンツの意味を理解するのに必要な音声読み上げの順序を保ちながら、ユーザエージェントがコンテンツの代替表現を提供できるようにすることである。意味のあるコンテンツの少なくとも一つの順序がプログラムによる解釈が可能であることが重要である。この達成基準を満たしていないコンテンツは、支援技術がそのコンテンツを正しくない順序で読み上げたり、代替スタイルシート又はその他の書式変更が適用されたりしたときに、利用者を困惑あるいは混乱させてしまう恐れがある。</p>
             <p>コンテンツの並び順を変更すると、コンテンツの意味に影響を及ぼす場合、その順序には<em>意味</em>がある。例えば、あるページに二つの独立した記事がある場合、それらの記事の相対的な順序がそれぞれの意味に影響を及ぼす可能性はない。そのような状況においては、それらの記事自体には意味のあるシーケンスがあるかもしれないが、それらの記事が入っているテキストコンテナには意味のあるシーケンスはないかもしれない。</p>
-            <p>ある要素のプログラムが持つ意味が、そのコンテンツに意味のあるシーケンスがあるかどうかを定義している。例えば、HTML では、テキストには常に意味のあるシーケンスがある。テーブル及び順序付きリストには意味のあるシーケンスがあるが、順序なしリストには意味のあるシーケンスがない。</p>
+            <p>いくつかの要素のセマンティクスは、その要素のコンテンツが意味のあるシーケンスがあるかどうかを定義する。例えば、HTML では、テキストには常に意味のあるシーケンスがある。テーブル及び順序付きリストには意味のあるシーケンスがあるが、順序なしリストには意味のあるシーケンスがない。</p>
```

Comment:
>The semantics of some elements define whether or not their content is a meaningful sequence. 

シーケンスがある → シーケンスである かも

```suggestion
            <p>いくつかの要素のセマンティクスは、その要素のコンテンツが意味のあるシーケンスであるかどうかを定義する。例えば、HTML では、テキストには常に意味のあるシーケンスがある。テーブル及び順序付きリストには意味のあるシーケンスがあるが、順序なしリストには意味のあるシーケンスがない。</p>
```